src/views/guestLayout.js

```diff
@@ -42,7 +42,7 @@ const guestLayout = data => `
 
     <!-- Google tag (gtag.js) -->
     <script nonce='${data.nonce}'>
-      if (!navigator.doNotTrack || navigator.doNotTrack !== '1') {
+      if (navigator.doNotTrack !== '1') {
```

Comment:
Huh! So I am thoroughly testing for both guest and main layout with:

- Firefox (with and without DNT)
- Chrome (with and without DNT)
- Safari (no DNT)

In the console, Safari evaluates:
```
> navigator.doNotTrack !== '1'
< true
```

But it seems to have the opposite effect when I load the test page, it's very odd.

Reply:
As an aside, this is super tedious to test. I am sure we can come up with some way to signal whether this DNT switch is working in a playwright test, I'll file a ticket in the backlog.